src/Features/Core/Portable/MetadataAsSource/AbstractMetadataAsSourceService.DocCommentFormatter.cs

```diff
@@ -30,15 +31,13 @@ internal class DocCommentFormatter
 
             internal static ImmutableArray<string> Format(IDocumentationCommentFormattingService docCommentFormattingService, DocumentationComment docComment)
```

Comment:
this whole system realizes a full array of strings, just to make more strings (using the VB or C# comment for `//` or `'`).  better would be to pass in that comment prefix and generate the final strings as is.  but i din't want to churn too much here.